core/src/components/card/card.tsx

```diff
@@ -91,11 +100,12 @@ export class Card implements ComponentInterface, AnchorInterface, ButtonInterfac
     if (!clickable) {
       return [<slot></slot>];
     }
-    const { href, routerAnimation, routerDirection } = this;
+    const { href, routerAnimation, routerDirection, inheritedAttributes } = this;
     const TagType = clickable ? (href === undefined ? 'button' : 'a') : ('div' as any);
+    const { ['aria-label']: ariaLabel } = inheritedAttributes;
     const attrs =
       TagType === 'button'
-        ? { type: this.type }
+        ? { type: this.type, ['aria-label']: ariaLabel }
```

Comment:
Wouldn't we want to do the following instead?

```js
<TagType
  {...attrs}
  class="card-native"
  part="native"
  disabled={this.disabled}
  onClick={(ev: Event) => openURL(href, ev, routerDirection, routerAnimation)}
  {...inheritedAttributes}
>
```

The current code would only apply the custom label if `TagType === 'button'` and would miss when `TagType === 'a'`.

Reply:
Applying the `aria-label` regardless of the `TagType` makes sense to me, I can make that change. I was cautious to append all values of `inheritedAttributes`, since we could potentially reflect attributes to different target elements in the component's template. 

For `ion-card`, I think this wouldn't matter, as the template is relatively simple and flat. 

`ion-item` does have more complexity in its template. 

I can reflect all of them for now and try to document in a way that alerts devs to this possible scenario.